src/settings/settingsManager.ts

```diff
@@ -45,12 +46,26 @@ export class SettingsManager {
     }
   }
 
+  public getRootSlpPath(): string {
+    return this.appSettings.settings?.rootSlpPath as string;
+  }
+
+  public getUseMonthlySubfolders(): boolean {
+    return this.appSettings.settings?.useMonthlySubfolders as boolean;
```

Comment:
Again you should be handling the undefined case instead of using optional chaining and type casting.